pkgs/by-name/wa/waagent/package.nix

```
@@ -63,14 +65,24 @@ python.pkgs.buildPythonApplication rec {
 
   dontWrapPythonPrograms = false;
 
-  meta = {
+  passthru = {
+    tests = {
```

```suggestion
    tests = lib.optionalAttrs stdenv.hostPlatform.isLinux {
```
These prevents OfBorg from attempting to evaluate and build this test on non-Linux platforms (where it will never work)

---

In the latest iteration I added `meta.platforms = lib.platforms.linux;` to make this package Linux-only, as the project does not support Darwin officially.

---

`passthru.tests` are still evaluated on Darwin regardless IIRC